django_cloud_deploy/tool_requirements.py

```diff
@@ -129,13 +129,26 @@ def check(cls):
         Raises:
             MissingRequirementError: If the requirement is not found.
         """
-        if not shutil.which('gcloud'):
-            download_link = (
-                'https://cloud.google.com/sdk/docs/downloads-interactive')
-            msg = ('Please download Google Cloud SDK '
-                   'from {}'.format(download_link))
+        if shutil.which('gcloud'):
+            return None
+
+        # Default paths
+        gcloud_config_path = os.path.expanduser('~/.config/gcloud')
+        gcloud_sdk_path = os.path.expanduser('~/.google-cloud-sdk/gcloud')
```

Comment:
It should be '~/google-cloud-sdk/gcloud'. (No dot before google-cloud-sdk).

Also, if gcloud is installed with apt-get, then the path of gcloud is '/usr/bin/gcloud'. I am not sure whether this matters.

Reply:
ty, good catch!